source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3647,6 +3675,10 @@ export class TextInput extends Input {
             result = document.createElement("input");
             result.className = this.hostConfig.makeCssClassName("ac-input", "ac-textInput");
             result.type = Enums.InputTextStyle[this.style].toLowerCase();
+
+            if (this.inputStyle !== null && this.inputStyle === Enums.InputStyle.ReadWrite && !this.inlineAction) {
+                result.classList.add(this.hostConfig.makeCssClassName("ac-textInput-readWriteStyle"));
```

Comment:
You shouldn't use class names that are specific to each input type. So instead of `ac-textInput-readWriteStyle` you should use `ac-textInput ac-inputStyle-readWrite`. This way you can have generic code that adds the `ac-inputStyle-readWrite` class to any input regardless of its actual type.